CoreFoundation/Base.subproj/CoreFoundation_Prefix.h

```diff
@@ -201,6 +201,12 @@ CF_INLINE uint64_t mach_absolute_time() {
 #define strtol_l(a,b,c,locale) strtol(a,b,c)
 
 #define fprintf_l(a,locale,b,...) fprintf(a, b, __VA_ARGS__)
+
+CF_INLINE int flsl( long mask ) {
+    int idx = 0;
+    while (mask != 0) mask = (unsigned long)mask >> 1, idx++;
```

Comment:
I would suggest removing the comma and putting this in braces to make it more obvious what this is doing.

Reply:
Done.

Reply:
Done